main.py

```
@@ -90,12 +101,13 @@ def main():
-        launch_bartender_interface(
-            handle_input_fn=handle_input_with_deps,
-            clear_state_fn=clear_chat_state,
-            share=True,
-            debug=True
-        )
+        try:
+            interface = launch_bartender_interface(
+                handle_input_fn=handle_input_with_deps,
+                clear_state_fn=clear_chat_state
+            )
+        except Exception as ui_err:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)